### PR TITLE
Task/3812 modal font size and check icon weight fix

### PR DESCRIPTION
**Pull request by tiwariav**

### CODE REVIEW

**Comment by verysanghee** [comment]
This PR works well except for my dev testing mail account. With the same card number and email address I've been using countless times, I received "invalid parameters" error on confirm order page(which sended me back to billing info page) and could never get through.

File: hosting/static/hosting/js/virtual_machine_detail.js [hunk 1]
```diff
@@ -90,26 +89,28 @@ $(document).ready(function() {
...
+                    $('#createvm-modal-done-btn').removeClass('hide');
                 }
             },
             error: function (xmlhttprequest, textstatus, message) {
                     fa_icon = $('.modal-icon > .fa');
-                    fa_icon.attr('class', 'fa fa-times');
-                    $('.modal-header > .close').attr('class', 'close');
-                    $('.modal-text').addClass('hide');
+                    fa_icon.attr('class', 'fa fa-close');
+                    // $('.modal-header > .close').attr('class', 'close');
```
Comment (pcoder) [inline hosting/static/hosting/js/virtual_machine_detail.js]: Let's remove the commented lines if not used.

File: hosting/static/hosting/js/virtual_machine_detail.js [hunk 2]
```diff
@@ -90,26 +89,28 @@ $(document).ready(function() {
...
                 if (data.status === true) {
                     fa_icon = $('.modal-icon > .fa');
-                    fa_icon.attr('class', 'fa fa-check');
-                    $('.modal-header > .close').attr('class', 'close');
+                    fa_icon.attr('class', 'checkmark');
+                    // $('.modal-header > .close').removeClass('hidden');
                     $('#createvm-modal-title').text(data.msg_title);
                     $('#createvm-modal-body').text(data.msg_body);
-                    $('#createvm-modal').on('hidden.bs.modal', function () {
-                        window.location = data.redirect;
```
Comment (pcoder) [inline hosting/static/hosting/js/virtual_machine_detail.js]: In the PR #493 which is yet to be merged, we seem to be using the same modal both for landing (user not logged in) and hosting (user logged in) cases. First the first case: on successful vm creation, `data.redirect` contains url to datacenterlight's index page, while for the logged in users, we take the user to his/her virtual machines page. I think the first case may not be covered in this PR. So, to be generic for both logged in and not logged in cases, I propose that we reuse `data.redirect`.

Comment (tiwariav) [reply hosting/static/hosting/js/virtual_machine_detail.js]: ok, didn't know that, reverting this back 👍

**Comment by tiwariav** [comment]
now, on success, `data.redirect` is set as the `href` attribute for the `<a>` tag.

**Pull request merged**